### PR TITLE
Release notes 0.19

### LINKED ISSUE #3024: Release notes blog for knative v0.19
Write the release notes blog for knative v0.19

This will require all major components to be cut for v0.19
- serving
- eventing
- eventing-contrib (now split into knative-sandbox)
- operator
- cli

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #3024 

[Rendered version](https://github.com/csantanapr/knative-docs/blob/release-notes-0.19/blog/releases/announcing-knative-v0-19-release.md)

### CODE REVIEW

**Comment by csantanapr** [comment]
What's left is Client and Operator.
@rhuss  @houshengbo  let me know when you have release notes for client and operator.

File: blog/releases/announcing-knative-v0-19-release.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+### Thank you contributors v0.19
+
+- [@antoineco](https://github.com/antoineco)
+- [@daisy-ycguo](https://github.com/daisy-ycguo)
+- [@danielhelfand](https://github.com/danielhelfand)
+- [@dprotaso](https://github.com/dprotaso)
+- [@dsimansk](https://github.com/dsimansk)
+- [@eclipselu](https://github.com/eclipselu)
+- [@ian-mi](https://github.com/ian-mi)
```
Comment (rhuss) [inline blog/releases/announcing-knative-v0-19-release.md:228]:
```suggestion
- [@ian-mi](https://github.com/ian-mi)
- [@itsmurugappan](https://github.com/itsmurugappan)
```

Not so obvious from the commits, but @itsmurugappan did quite some contributions. Thanks !

Comment (rhuss) [reply blog/releases/announcing-knative-v0-19-release.md:228]: I think we should think about to call out contributors more prominently in the client release notes.

**Comment by csantanapr** [comment]
CLI is done, Operator I'm waiting to be cut

File: blog/releases/announcing-knative-v0-19-release.md (hunk 1 above)
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-19-release.md:228]: I have been doing small changes from release notes to release notes, this time I included the list of contributors to give thanks.

Comment (csantanapr) [reply blog/releases/announcing-knative-v0-19-release.md:228]: @rhuss sorry I missed @itsmurugappan I thought I got everyone reading the PRs.

Comment (csantanapr) [reply blog/releases/announcing-knative-v0-19-release.md:228]: Ah I know what happened PR was missing from the changelog https://github.com/knative/client/blob/master/CHANGELOG.adoc#v0190-2020-11-11

That's what I used to extract the github ids for client

**Comment by csantanapr** [comment]
@evankanderson @abrennan89 could you take a look this is ready now.

File: blog/releases/announcing-knative-v0-19-release.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+A new version of Knative is now available across multiple components.
+Follow the instructions in the documentation [Installing Knative](https://knative.dev/docs/install/) for the respective component.
+
+#### Table of Contents
+- [Serving v0.19](#serving-v019)
+- [Eventing v0.19](#eventing-v019)
+- Eventing Extensions
+    - [Eventing Kafka Broker v0.19](#eventing-kafka-broker-v019)
+    - [Eventing Gitlab v0.19](#eventing-gitlab-v019)
```
Comment (evankanderson) [inline blog/releases/announcing-knative-v0-19-release.md:23]: Did you also want the RabbitMQ bits in the TOC?

**Review by evankanderson — COMMENTED** [review]
/lgtm
/hold

You can `/hold cancel` if you don't need to fix the table of contents issue, otherwise I can take another look tomorrow morning.

File: blog/releases/announcing-knative-v0-19-release.md (hunk 2 above)
Comment (csantanapr) [reply blog/releases/announcing-knative-v0-19-release.md:23]: Yes good catch, I missed rabbit mq I Will add

Comment (csantanapr) [reply blog/releases/announcing-knative-v0-19-release.md:23]: @evankanderson thanks for the review I fix the TOC, but will keep it on hold until the docs are released for v0.19 are public

**Comment by csantanapr** [comment]
@evankanderson @RichieEscarez this is ready now with docs published now

**Review by evankanderson — COMMENTED** [review]
/lgtm

**Pull request merged**